### PR TITLE
feat(CTCP-3159): add lrn in subheading for review cancellation errors

### PR DESCRIPTION
**Pull request by malaw-hmrc**

### CODE REVIEW

File: test/views/arrival/P5/ReviewUnloadingRemarkErrorsP5ViewSpec.scala [hunk 1]
```diff
@@ -66,12 +66,8 @@ class ReviewUnloadingRemarkErrorsP5ViewSpec extends CheckYourAnswersViewBehaviou
 
   "must render correct paragraph1 content" in {
```
Comment (Reece-Carruthers) [inline test/views/arrival/P5/ReviewUnloadingRemarkErrorsP5ViewSpec.scala:67]: Missing the test for the caption in this view

File: test/views/departure/testOnly/RejectionMessageP5ViewSpec.scala [hunk 2]
```diff
@@ -65,12 +65,8 @@ class RejectionMessageP5ViewSpec extends CheckYourAnswersViewBehaviours with Gen
 
   "must render correct paragraph1 content" in {
```
Comment (Reece-Carruthers) [inline test/views/departure/testOnly/RejectionMessageP5ViewSpec.scala:66]: Missing the test for the caption in this view

File: test/views/departure/testOnly/ReviewCancellationErrorsP5ViewSpec.scala [hunk 3]
```diff
@@ -44,6 +44,8 @@ class ReviewCancellationErrorsP5ViewSpec extends CheckYourAnswersViewBehaviours
 
   behave like pageWithBackLink()
 
+  behave like pageWithCaption(s"LRN: $lrn")
```
Comment (Reece-Carruthers) [inline test/views/departure/testOnly/ReviewCancellationErrorsP5ViewSpec.scala:47]: This is the test that should be in the other comment views

File: test/views/departure/testOnly/ReviewDepartureErrorsP5ViewSpec.scala [hunk 4]
```diff
@@ -67,11 +67,7 @@ class ReviewDepartureErrorsP5ViewSpec extends CheckYourAnswersViewBehaviours wit
   "must render correct paragraph1 content" in {
     assertSpecificElementContainsText(
       "paragraph-1-prefix",
```
Comment (Reece-Carruthers) [inline test/views/departure/testOnly/ReviewDepartureErrorsP5ViewSpec.scala:69]: Caption test in this file too I believe

File: conf/messages.en [hunk 5]
```diff
@@ -437,7 +437,7 @@ departure.declaration.errors.message.hyperlink = Create another departure declar
 cancellation.notification.errors.message.title   = Cancellation errors
 cancellation.notification.errors.message.heading = Cancellation errors
 cancellation.notification.errors.message.noerrors = There are one or more errors with the cancellation of departure declaration {0}.
```
Comment (Reece-Carruthers) [inline conf/messages.en:439]: I think this can be deleted if it's not being used

File: conf/messages.en [hunk 6]
```diff
@@ -437,7 +437,7 @@ departure.declaration.errors.message.hyperlink = Create another departure declar
...
-cancellation.notification.errors.message.elevenpluserrors = There are a number of errors with the cancellation of departure declaration {0}.
+cancellation.notification.errors.message.elevenpluserrors = There are one or more errors with the cancellation of this declaration.
```
Comment (Reece-Carruthers) [inline conf/messages.en:440]: The key could be changed to just be something like `cancellation.notification.errors.message` to be more reflective of the actual content

File: conf/messages.en [hunk 7]
```diff
@@ -447,7 +447,7 @@ cancellation.notification.customsOfficeContact.teleNotAvailAndOfficeNameNotAvail
 arrival.notification.errors.message.title   = Notification errors
 arrival.notification.errors.message.heading = Notification errors
 arrival.notification.errors.message.noerrors = There are one or more errors in arrival notification {0} that cannot be amended.
-arrival.notification.errors.message.elevenpluserrors = There are a number of errors in arrival notification {0}.
+arrival.notification.errors.message.elevenpluserrors = There are one or more errors in this notification that cannot be amended. Make a new notification with the right information.
```
Comment (Reece-Carruthers) [inline conf/messages.en:450]: Key could be changed

File: conf/messages.en [hunk 8]
```diff
@@ -447,7 +447,7 @@ cancellation.notification.customsOfficeContact.teleNotAvailAndOfficeNameNotAvail
 arrival.notification.errors.message.title   = Notification errors
 arrival.notification.errors.message.heading = Notification errors
 arrival.notification.errors.message.noerrors = There are one or more errors in arrival notification {0} that cannot be amended.
```
Comment (Reece-Carruthers) [inline conf/messages.en:449]: I think this can be deleted

Comment (malaw-hmrc) [reply conf/messages.en]: Done

File: conf/messages.en (hunk 5 above)
Comment (malaw-hmrc) [reply conf/messages.en:439]: I think this is being used Reece in CancellationNotificationErrorsP5ViewModel, so I am not sure if we can delete it... 
    if (noErrors) {
      messages("cancellation.notification.errors.message.noerrors", lrn)
    }

Comment (Reece-Carruthers) [reply conf/messages.en:439]: Hmm true, maybe leave them in for now. There is two tickets to address these changes where this should get taken care of anyways. (CTCP-3284, CTCP-3289)

Comment (malaw-hmrc) [reply conf/messages.en:439]: Resolved, will been taken care in another ticket

File: app/viewModels/P5/departure/DepartureStatusP5ViewModel.scala [hunk 9]
```diff
@@ -309,4 +308,9 @@ object DepartureStatusP5ViewModel {
       DepartureStatusP5ViewModel("movement.status.P5.guaranteeWrittenOff", actions = Nil)
   }
 
+  private def errorsActionText(errors: Seq[String]): String = if (errors.length == 1) {
+    "viewError"
+  } else {
+    "viewErrors"
+  }
```
Comment (MatthewAlexanderBeck) [inline app/viewModels/P5/departure/DepartureStatusP5ViewModel.scala:315]: This look fine however can we add some updated unit tests in the DepartureStatusP5ViewModelSpec please

**Review by MatthewAlexanderBeck — APPROVED** [review]

**Review by srinivasulumunagalahmrc — APPROVED** [review]

**Comment by vajarova** [comment]
Overriden by another branch CTCP-3159-remastered, closing this

**Pull request closed without merging**